lib/logitech_receiver/base.py

```diff
@@ -95,11 +95,13 @@ def receivers():
         for d in _hid.enumerate(receiver_usb_id):
             yield d
 
```

Comment:
It would be nice to have this style change not be a separate commit but instead be part of the underlying commit.